dash/backend/src/config/metrics.ts

```diff
@@ -0,0 +1,8 @@
+export default {
+  disableEndpoint: (process.env.METRICS_DISABLE_ENDPOINT || 'false').toString().toLowerCase() === 'true',
+  secureEndpoint: (process.env.METRICS_SECURE_ENDPOINT || 'false').toString().toLowerCase() === 'true',
```

Comment:
The rest of the config uses 1 or 0 to determine boolean config, this should be made consistent.

See database.migrations.enabled, email.system.enableErrorReportEmail, rabbitMQ.enabled for patterns (Annoyingly, they are all slightly different)